### DISCUSSION TITLE
Bring visual alignment between developer and configuration menus

### DISCUSSION THREAD

**Discussion by poldim** [home-assistant/frontend#11295] [Other feature requests]
## Bring visual alignment between developer and configuration menus 

It would be great to have a similar look and feel between the Developer and the Configuration Menus. It currently looks like these two were designed by completely different teams (which could be entirely possible).

While being cognizant that the overall design should not make too many clicks to get to the services tab, but this would only add one click to get to the services screen

**TODAY:**
![image](https://user-images.githubusercontent.com/16869713/93907542-4e85ef00-fcb2-11ea-831c-6520736bc57b.png)

**POSSIBLE:**
![image](https://user-images.githubusercontent.com/16869713/93907631-6f4e4480-fcb2-11ea-867a-ba4417d98df0.png)

## Additional information

I have the Chrome Inspect window open but can’t copy the HTML code because of what I think are these shadow-root errors.  If anyone knows how to fix this, LMK.

**Comment by maykar** [comment]
@bramkragten feel free to reopen this one, I mistakenly thought this was more about the alignment between the elements in the header of dev tools rather than a redesign of the page itself. My apologies.
